src/connection/utils.ts

```diff
@@ -12,18 +12,18 @@ export function getIsInjected(): boolean {
   return Boolean(window.ethereum)
 }
 
+export function getIsBraveWallet(): boolean {
+  return window.ethereum?.isBraveWallet ?? false
+}
+
 export function getHasMetaMaskExtensionInstalled(): boolean {
-  return window.ethereum?.isMetaMask ?? false
+  return (window.ethereum?.isMetaMask ?? false) && !getIsBraveWallet()
```

Comment:
We could link the documentation here, as it's hard to figure out what's going on. Without it, it feels like we are prioritizing the Brave wallet inside the Brave browser.

Furthermore, `getHasMetaMaskExtensionInstalled`, does not seem to be semantically correct anymore. The user could have MetaMask installed as an extension, but prioritized the Brave browser in their settings. I was wondering if `isMetaMaskWallet` or something along those lines makes more sense now 🤔

Reply:
You are right, that name could be somewhat less confusing!